index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="de">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Emilie´s personal website</title>
+</head>
+
+<body>
+    <header>
+        <h1>Emilie Lokietz</h1>
+        <nav>
+            <a href="#Kontakt">Kontakt</a>
+            <a href="#Biographie">Biographie</a>
+            <a href="">Mein Hund</a> <!-- da ist die Idee auf eine neue Website überzuleiten mit einer Galerie von den Fotos von meinem Hund-->
+        </nav>
+    </header>
+
+    <section>
+        <img src="hund.jpg"
+            alt="Foto" > <!--Das Photo was bei mir im Ordner ist wird nicht geladen...-->
```

Comment:
Just a little hint: remove the space at the end behind the alt-Text ;-)

Reply:
thanks I will